gsplat/strategy/default.py

```diff
@@ -202,19 +207,28 @@ def _update_state(
         info: Dict[str, Any],
         packed: bool = False,
     ):
-        for key in ["means2d", "width", "height", "n_cameras", "radii", "gaussian_ids"]:
+        for key in [
+            "means2d",
```

Comment:
since we check for `key_for_gradient`, we could remove the check for `means2d` right?